### DISCUSSION TITLE
Problema Duckdns con modem fibra Iliad

### DISCUSSION THREAD

**Discussion by Franco2255** [indomus/forum#3067] [HUB: Home Assistant]
Ciao a tutti, da ieri sono passato alla fibra di Iliad e non riesco più a connettermi da remoto con app Home Assistant.
il modem di Iliad non permette di aprire le porte al di sotto di 16384 è solo un problema mio? 
Qualcuno con fibra Iliad ha trovato la soluzione?
Grazie

**Comment by Scialla** [comment]
Mi sa che non hai solo quel problema, iliad non fornisce IPV4 ma solo IPV6...

Ti consiglio di seguire la guida con cloudflare
https://indomus.it/guide/collegarsi-da-remoto-a-home-assistant-via-reti-provider-tradizionali-o-che-usino-nat-via-cloudflare/

**Reply by Franco2255** [reply]
Grazie per i suggerimenti.
Risolto Iliad mi ha aperto IPV4 con le relative porte.

**Reply by Scialla** [reply]
Sei riuscito a farti dare IPV4 in esclusiva o condiviso?
Quali porte?
Hai fatto richiesta in assistenza?

Giusto per lasciare traccia di come hai risolto per chi ne avesse bisogno, anche perché Iliad sembra essere la fibra più economica sul mercato

**Reply by Franco2255** [reply]
dovrebbe essere in esclusiva, non posso assicuralo.
per quanto riguarda la linea la wifi  con tim andavo al massimo a 300 con iliad vado a 1000

**Reply by snakuzzo** [reply]
Non è in esclusiva. Gli IPv4 pubblici con iliad sono condivisi. Probabilmente sono i range di porte utilizzabili per il port forwarding ad essere esclusivi

**Reply by Franco2255** [reply]
quando ho parlato col tecnico iliad ma ha detto che mi cambiava ip pero non ha specificato se era condiviso o no.
da quando è cambiato l'ip duckdns ha ripreso a funzionare

**Reply by snakuzzo** [reply]
Stavo leggendo in giro...come dicevo prima, di default iliad dà IPv4 pubblico condiviso da un massimo di 4 utenti e ognuno di essi ha a disposizione 1/4 delle porte disponibili da utilizzare per i propri servizi.
Premesso che, volendo, si potrebbe usare IPv6 al posto di IPv4, pare che chiamando il call center ci si possa far assegnare un IPv4 full stack, quindi un IPv4 pubblico totalmente dedicato al singolo utente.

**Reply by texindustries** [reply]
Sto pensando pure io di passare alla fibra Iliad, provengo da fastweb con FTTC, anni fa appunto mi avevo fatto dare un IP pubblico. Con Iliad alla fine siete riusciti a risolvere l'apertura delle porte?
Grazie

**Reply by snakuzzo** [reply]
Non c'è alcun problema di apertura porte con iliad.
Iliad dà IPv4 pubblico condiviso. Nel senso che uno stesso ip è condiviso da 4 clienti. 

Vengono assegnate circa 16000 porte a cliente da poter utilizzare...e sono visibili direttamente sul router di Iliad.

**Reply by snakuzzo** [reply]
Resta il fatto che la soluzione migliore in assoluto, che si abbia ip pubblico oppure no, è il tunnel con cloudflare.

**Reply by texindustries** [reply]
Concordo... il passaggio a iliad volevo farlo in modo quasi trasparente: purtroppo par mancanza di tempo, sto usando duckdns e nginx. a questo punto fatto il passaggio, chiederò un ip full stack.
Appena avrò più tempo ricomincerò un po tutta la configurazione di HA da zero, oggi ho ancora un bel po di codice vecchio che si potrebbe migliorare di molto.

intanto grazie a tutti

**Comment by 50494554524F** [comment]
prova con l'addon tor

**Comment by hippos78** [comment]
Io ho risolto il problema, senza dover utilizzare nessun DNS, semplicemente creando un reindirizzamento della porta dentro le impostazioni del modem di iliad e accedo a home assistant ovunque da esterno senza problemi.
